packages/edit-site/src/components/editor/index.js

```diff
@@ -269,7 +269,7 @@ export default function Editor( { isLoading } ) {
 								<BlockBreadcrumb
 									rootLabelText={
 										postWithTemplate &&
-										renderingMode !== 'template-only'
+										currentPostType !== 'wp_template'
 											? __( 'Page' )
 											: __( 'Template' )
 									}
```

Comment:
This one seems like a bug. In the site editor, if you try editing a pattern it shows 'Template' as the breadcrumb.

Instead I think it should show the post type label like the post editor does.

Reply:
Yeh, the post type label makes more sense here. Done.